gum/arch-x86/gumx86relocator.c

```diff
@@ -591,6 +591,8 @@ gum_x86_relocator_rewrite_if_rip_relative (GumX86Relocator * self,
     X86_REG_RSI, X86_REG_RDI
   };
   gint rip_reg_index, i;
+  GumAddress  address;
+  gssize offset;
```

Comment:
These two should be right after `is_rip_relative`, as per [Local variables are listed in chronological order of usage](https://github.com/frida/frida/blob/643acca36d13eb5f33ab687d62ac9fb116722608/CONTRIBUTING.md#rule-21).

Reply:
Yeah. I remembered that rule. But I put them immediately before `other_reg` since it was the next local used. However, it seems that one might be out of order in relation to `cpu_regs`, `cs_regs` and `rip_reg_index`?

Reply:
Ah yes, seems this function has degraded a bit over the years — I probably did a bad job with a previous review. :/

Reply:
No worries. No code is perfect. Especially not mine!